api/mesh/v1alpha1/dataplane.proto

```diff
@@ -163,6 +168,9 @@ message Dataplane {
 
     // TransparentProxying describes configuration for transparent proxying.
     TransparentProxying transparent_proxying = 4;
+
+    // Admin contains configuration related to Envoy Admin API
+    Admin admin = 8;
```

Comment:
No, it's actually 8, maybe I should've added `admin` to the beginning of `Networking` message:
```
message Networking {

  // Public IP on which the dataplane is accessible in the network.
  string address = 5;

  reserved 6; // Formerly ingress mode message, see #3435

  // In some situation, dataplane resides in a private network and not
  // reachable via 'address'. advertisedAddress is configured with public
  // routable address for such dataplane so that other dataplanes in the mesh
  // can connect to it over advertisedAddress and not via address
  // Note: Envoy binds to the address not advertisedAddress
  string advertisedAddress = 7;
...  
```